elixir/apps/web/lib/web/live/sign_up.ex

```diff
@@ -0,0 +1,228 @@
+      </div>
+      <div class="text-center">
+        <div class="px-12">
+          <table class="border-collapse table-fixed w-full text-sm">
+            <tbody>
+              <tr>
+                <td class={~w[border-b border-slate-100 dark:border-slate-700
+                              p-4 pl-8 text-gray-900 dark:text-white]}>
+                  Account Name:
+                </td>
+                <td class={~w[border-b border-slate-100 dark:border-slate-700
+                              p-4 pl-8 text-gray-900 dark:text-white]}>
+                  <%= @account.name %>
+                </td>
+              </tr>
+              <tr>
+                <td class={~w[border-b border-slate-100 dark:border-slate-700
+                              p-4 pl-8 text-gray-900 dark:text-white]}>
+                  Account Slug:
+                </td>
+                <td class={~w[border-b border-slate-100 dark:border-slate-700
+                              p-4 pl-8 text-gray-900 dark:text-white]}>
+                  <%= @account.slug %>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+      <div class="text-base leading-7 text-center text-gray-900 dark:text-white">
+        <div>
+          Sign In URL
+        </div>
+        <div>
+          <.link
+            class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
+            navigate={~p"/#{@account.slug}/sign_in"}
+          >
+            <%= "#{Web.Endpoint.url()}/#{@account.slug}/sign_in" %>
+          </.link>
+        </div>
+      </div>
+    </div>
+    """
+  end
+
+  def sign_up_form(assigns) do
+    ~H"""
+    <h3 class="text-center text-m font-bold leading-tight tracking-tight text-gray-900 sm:text-xl dark:text-white">
+      Sign Up Now
+    </h3>
+    <.simple_form for={@form} class="space-y-4 lg:space-y-6" phx-submit="submit" phx-change="validate">
+      <.inputs_for :let={account} field={@form[:account]}>
+        <.input
+          field={account[:name]}
+          type="text"
+          label="Account Name"
+          placeholder="Enter an Account Name here"
+          required
+          phx-debounce="300"
+        />
+        <.input field={account[:slug]} type="hidden" />
+      </.inputs_for>
+
+      <.inputs_for :let={actor} field={@form[:actor]}>
+        <.input
+          field={actor[:name]}
+          type="text"
+          label="Your Name"
+          placeholder="Enter your name here"
+          required
+          phx-debounce="300"
+        />
+        <.input field={actor[:type]} type="hidden" />
+      </.inputs_for>
+
+      <.input
+        field={@form[:email]}
+        type="text"
+        label="Email"
+        placeholder="Enter your email here"
+        required
+        phx-debounce="300"
+      />
+
+      <:actions>
+        <.button phx-disable-with="Creating Account..." class="w-full">
+          Create Account
+        </.button>
+      </:actions>
```

Comment:
Sadly I think we'll need a snippet of legalese here, something along the lines of:

```suggestion
      </:actions>
      <p class="text-xs">
        By signing up you agree to our <.link href="https://www.firezone.dev/terms">Terms of Use</.link>.
      </p>
```

Will probably add more polish later.

Reply:
👍 Will get this added in.

Reply:
This has been added in to the page.